packages/vant/src/image/Image.tsx

```diff
@@ -29,10 +29,13 @@ const [name, bem] = createNamespace('image');
 
 export type ImageFit = 'contain' | 'cover' | 'fill' | 'none' | 'scale-down';
 
+export type ImagePosition = 'center' | 'top' | 'right' | 'bottom' | 'left';
+
 const imageProps = {
   src: String,
   alt: String,
   fit: String as PropType<ImageFit>,
+  position: String as PropType<ImagePosition>,
```

Comment:
maybe should be `string` type to support `10px`

Reply:
just updated to align with `object-position`